[2024-12-21T10:23:24.080+0530] {processor.py:186} INFO - Started process (PID=2222) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:23:24.081+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:23:24.083+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:23:24.082+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:23:25.180+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:23:25.480+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:23:25.479+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:23:25.523+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:23:25.517+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:23:25.524+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:23:25.524+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:23:25.596+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.523 seconds
[2024-12-21T10:25:46.243+0530] {processor.py:186} INFO - Started process (PID=2584) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:25:46.245+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:25:46.246+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:25:46.246+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:25:47.171+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:25:47.495+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:25:47.494+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:25:47.528+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:25:47.526+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:25:47.528+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:25:47.528+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:25:47.553+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.316 seconds
[2024-12-21T10:27:08.045+0530] {processor.py:186} INFO - Started process (PID=2800) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:27:08.047+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:27:08.049+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:27:08.049+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:27:08.958+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:27:09.185+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:27:09.185+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:27:09.215+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:27:09.213+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:27:09.215+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:27:09.215+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:27:09.241+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.202 seconds
[2024-12-21T10:28:20.453+0530] {processor.py:186} INFO - Started process (PID=3000) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:28:20.454+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:28:20.456+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:28:20.456+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:28:21.415+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:28:21.744+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:28:21.743+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:28:21.791+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:28:21.789+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:28:21.792+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:28:21.792+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:28:21.825+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.377 seconds
[2024-12-21T10:29:40.409+0530] {processor.py:186} INFO - Started process (PID=3214) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:29:40.411+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:29:40.416+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:29:40.416+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:29:41.803+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:29:42.177+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:29:42.177+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:29:42.224+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:29:42.222+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:29:42.225+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:29:42.224+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:29:42.404+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 2.002 seconds
[2024-12-21T10:31:19.472+0530] {processor.py:186} INFO - Started process (PID=3449) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:31:19.473+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:31:19.475+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:31:19.475+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:31:20.719+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:31:21.051+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:31:21.050+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:31:21.268+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:31:21.228+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:31:21.270+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:31:21.269+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:31:21.476+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 2.008 seconds
[2024-12-21T10:47:45.947+0530] {processor.py:186} INFO - Started process (PID=4069) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:47:45.951+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:47:45.954+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:47:45.953+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:47:47.825+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:47:48.601+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:47:48.600+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:47:48.706+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:47:48.701+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:47:48.707+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:47:48.707+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:47:48.807+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 2.869 seconds
[2024-12-21T10:50:06.333+0530] {processor.py:186} INFO - Started process (PID=4518) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:50:06.335+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:50:06.336+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:50:06.336+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:50:07.440+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:50:07.759+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:50:07.759+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:50:07.820+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:50:07.813+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:50:07.821+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:50:07.821+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:50:07.867+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.540 seconds
[2024-12-21T10:51:54.195+0530] {processor.py:186} INFO - Started process (PID=4899) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:51:54.196+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:51:54.198+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:51:54.198+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:51:55.274+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:51:55.789+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:51:55.788+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:51:55.826+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:51:55.824+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:51:55.827+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:51:55.826+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:51:55.856+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.666 seconds
[2024-12-21T10:53:27.085+0530] {processor.py:186} INFO - Started process (PID=5224) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:53:27.089+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T10:53:27.093+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:53:27.092+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:53:28.891+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T10:53:29.226+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:53:29.226+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T10:53:29.266+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:53:29.264+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T10:53:29.267+0530] {logging_mixin.py:190} INFO - [2024-12-21T10:53:29.267+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T10:53:29.299+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 2.223 seconds
[2024-12-21T11:55:01.289+0530] {processor.py:186} INFO - Started process (PID=6587) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:55:01.292+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T11:55:01.294+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:55:01.294+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:55:02.250+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:55:02.513+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:55:02.513+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T11:55:02.550+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:55:02.545+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T11:55:02.551+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:55:02.550+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T11:55:02.619+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.335 seconds
[2024-12-21T11:57:04.034+0530] {processor.py:186} INFO - Started process (PID=7032) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:57:04.036+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T11:57:04.038+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:57:04.037+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:57:05.288+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:57:05.666+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:57:05.666+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T11:57:05.708+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:57:05.704+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T11:57:05.709+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:57:05.708+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T11:57:05.782+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.755 seconds
[2024-12-21T11:58:40.846+0530] {processor.py:186} INFO - Started process (PID=7381) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:58:40.849+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T11:58:40.851+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:58:40.850+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:58:42.077+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T11:58:42.342+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:58:42.342+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T11:58:42.384+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:58:42.377+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T11:58:42.385+0530] {logging_mixin.py:190} INFO - [2024-12-21T11:58:42.384+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T11:58:42.441+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.599 seconds
[2024-12-21T12:00:23.430+0530] {processor.py:186} INFO - Started process (PID=7749) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:00:23.432+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T12:00:23.437+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:00:23.437+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:00:25.088+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:00:25.588+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:00:25.587+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T12:00:25.665+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:00:25.637+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T12:00:25.666+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:00:25.666+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T12:00:25.722+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 2.299 seconds
[2024-12-21T12:01:58.728+0530] {processor.py:186} INFO - Started process (PID=8089) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:01:58.731+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T12:01:58.734+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:01:58.734+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:02:00.267+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:02:00.582+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:02:00.581+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T12:02:00.660+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:02:00.646+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T12:02:00.661+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:02:00.660+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T12:02:00.725+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 2.006 seconds
[2024-12-21T12:03:44.520+0530] {processor.py:186} INFO - Started process (PID=8453) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:03:44.522+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T12:03:44.526+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:03:44.525+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:03:46.728+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:03:47.121+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:03:47.120+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T12:03:47.165+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:03:47.160+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T12:03:47.166+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:03:47.165+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T12:03:47.233+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 2.721 seconds
[2024-12-21T12:05:34.418+0530] {processor.py:186} INFO - Started process (PID=8837) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:05:34.420+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T12:05:34.422+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:05:34.421+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:05:35.632+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T12:05:35.979+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:05:35.978+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T12:05:36.017+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:05:36.013+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T12:05:36.017+0530] {logging_mixin.py:190} INFO - [2024-12-21T12:05:36.017+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T12:05:36.083+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 1.671 seconds
[2024-12-21T22:32:12.573+0530] {processor.py:186} INFO - Started process (PID=15905) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:32:12.577+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T22:32:12.585+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:12.584+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:32:12.618+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:32:12.959+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:12.958+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T22:32:13.078+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:13.069+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T22:32:13.081+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:13.080+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T22:32:13.139+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 0.588 seconds
[2024-12-21T22:32:44.890+0530] {processor.py:186} INFO - Started process (PID=16157) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:32:44.894+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T22:32:44.902+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:44.901+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:32:44.930+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:32:45.166+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:45.165+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T22:32:45.397+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:45.386+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T22:32:45.399+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:32:45.398+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T22:32:45.486+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 0.618 seconds
[2024-12-21T22:33:15.687+0530] {processor.py:186} INFO - Started process (PID=16357) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:33:15.692+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T22:33:15.703+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:15.701+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:33:15.729+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:33:15.822+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:15.822+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T22:33:15.895+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:15.888+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T22:33:15.897+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:15.896+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T22:33:15.960+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 0.299 seconds
[2024-12-21T22:33:46.450+0530] {processor.py:186} INFO - Started process (PID=16560) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:33:46.454+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T22:33:46.461+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:46.460+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:33:46.481+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T22:33:46.603+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:46.602+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T22:33:46.687+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:46.680+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T22:33:46.690+0530] {logging_mixin.py:190} INFO - [2024-12-21T22:33:46.688+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T22:33:46.737+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 0.307 seconds
[2024-12-21T23:52:53.130+0530] {processor.py:186} INFO - Started process (PID=18018) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T23:52:53.134+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T23:52:53.141+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:52:53.139+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T23:52:53.159+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T23:52:53.310+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:52:53.309+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T23:52:53.468+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:52:53.418+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T23:52:53.470+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:52:53.469+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T23:52:53.517+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 0.405 seconds
[2024-12-21T23:53:24.132+0530] {processor.py:186} INFO - Started process (PID=18255) to work on /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T23:53:24.136+0530] {processor.py:914} INFO - Processing file /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py for tasks to queue
[2024-12-21T23:53:24.146+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:53:24.144+0530] {dagbag.py:588} INFO - Filling up the DagBag from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T23:53:24.174+0530] {processor.py:925} INFO - DAG(s) 'example_workday_timetable' retrieved from /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py
[2024-12-21T23:53:24.292+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:53:24.291+0530] {dag.py:3239} INFO - Sync 1 DAGs
[2024-12-21T23:53:24.364+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:53:24.358+0530] {dag.py:1182} ERROR - Failed to fetch run info after data interval None for DAG 'example_workday_timetable'
Traceback (most recent call last):
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/models/dag.py", line 1177, in next_dagrun_info
    info = self.timetable.next_dagrun_info(
  File "/Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/plugins/workday.py", line 83, in next_dagrun_info
    next_start = max(earliest, DateTime.combine(Date.today(), Time.min))
TypeError: can't compare offset-naive and offset-aware datetimes
[2024-12-21T23:53:24.367+0530] {logging_mixin.py:190} INFO - [2024-12-21T23:53:24.365+0530] {dag.py:4180} INFO - Setting next_dagrun for example_workday_timetable to None, run_after=None
[2024-12-21T23:53:24.428+0530] {processor.py:208} INFO - Processing /Users/jayesh/Desktop/Personal Work/Movie_data_pipeline/.venv/lib/python3.9/site-packages/airflow/example_dags/example_workday_timetable.py took 0.321 seconds
